index.js

```diff
@@ -1,3 +1,8 @@
+// Guard against document not being defined in non-browser environments.
+if (typeof global.document !== 'undefined') {
```

Comment:
Totally missed this but this if check is backwards.. I have fixed it in master. Please test PRs :) My oversight in not testing this better before merging.